float64[] positionLeft
float64[] positionRight
float64[] orientationLeft
float64[] orientationRight
float64[] gripperLeft
float64[] gripperRight
float64 pointTime